tests/api.c

```diff
@@ -4318,6 +4360,28 @@ static void test_wolfSSL_BN(void)
     #endif /* defined(OPENSSL_EXTRA) && !defined(NO_ASN) */
 }
 
+static void test_wolfSSL_ASN1_INTEGER_get(void){
+#if defined(OPENSSL_EXTRA) && !defined(NO_ASN)
+    ASN1_INTEGER ai;
+    long a;
+
+    AssertNull(a =wolfSSL_ASN1_INTEGER_get(NULL));
+
+    ai.data[0] = 0x02;
+    ai.data[1] = 0x01;
+    ai.data[2] = 0x03;
+
+    AssertNotNull(a =wolfSSL_ASN1_INTEGER_get(&ai));
```

Comment:
It would be a good idea to verify that "a" is the value that we expect here.  Please add checks against a known value.